pom.xml

```diff
@@ -136,6 +136,12 @@
 				<type>pom</type>
 				<scope>import</scope>
 			</dependency>
+			<!-- TODO: Why isn't the version taken from boot bom? -->
+			<dependency>
+				<groupId>org.springframework</groupId>
+				<artifactId>spring-core</artifactId>
+				<version>4.3.4.RELEASE</version>
```

Comment:
@dsyer any idea about this? I have to provide the `spring-core` version - otherwise the dependency is not found